### PR TITLE
Exam Returned Date - Adding Back End Components

### PR DESCRIPTION
**Pull request by akroon3r**
Client testing found that the return exam modal required a date field instead of a y/n dropdown. This PR contains the migration version file and model/schema/end-point changes required to add this field.

### CODE REVIEW

File: api/app/models/bookings/exam.py [hunk 1]
```diff
@@ -32,8 +32,9 @@ class Exam(Base):
     number_of_students = db.Column(db.Integer, nullable=True)
     exam_method = db.Column(db.String(15), nullable=False)
     deleted_date = db.Column(db.String(50), nullable=True)
-    exam_returned_ind = db.Column(db.Integer, nullable=False, default=0)
+    exam_returned_date = db.Column(db.String(256), nullable=True)
```
Comment (sjrumsby) [inline api/app/models/bookings/exam.py]: Should this be a db.DateTime?

File: api/migrations/versions/c7b76aecd1ca_.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+"""empty message
```
Comment (sjrumsby) [inline api/migrations/versions/c7b76aecd1ca_.py]: Can these two migrations be combined?

File: api/app/models/bookings/exam.py (hunk 1 above)
Comment (akroon3r) [reply api/app/models/bookings/exam.py]: fixed

File: api/migrations/versions/c7b76aecd1ca_.py (hunk 2 above)
Comment (akroon3r) [reply api/migrations/versions/c7b76aecd1ca_.py]: combined, pushed up.

**Pull request closed without merging**